docs/development-api/github.md

```diff
@@ -20,6 +20,56 @@ const buildName = GitHub.buildNameFromEnvironment();
 
 ---
 
+## pushFileChanges
+
+Pushes file changes from the local workspace to a specified GitHub branch.
+
+These file changes will appear as verified commits on GitHub.
+
+Please note: this will not update the local Git repository
```

Comment:
```suggestion
This function is roughly equivalent to `git push`,
but it will not update the local Git repository.
```

Reply:
I would suggest it's not similar since push would push commits and this would not?

Reply:
Perhaps maybe avoiding `push` might be wise? Would `upload` make more sense?

Reply:
Ah yes, I clearly wrote the wrong thing here. Will revert 🙇